packages/web3-core-method/src/index.js

```diff
@@ -253,16 +253,10 @@ Method.prototype._confirmTransaction = function (defer, result, payload) {
             }
```

Comment:
Can you please do this changes in a new branch an submit this PR against 1.0? Because it has nothing to do with the new ABI lib. Thanks :)